wrappers/ortools.rb

```diff
@@ -315,9 +315,12 @@ def solve(vrp, job, thread_proc = nil, &block)
           ),
           rests: vehicle.rests.collect{ |rest|
             OrtoolsVrp::Rest.new(
-              time_windows: rest.timewindows.collect{ |tw|
-                OrtoolsVrp::TimeWindow.new(start: tw.start, end: tw.end || 2**56)
-              },
+              # Ortools can only deal with one and exactly one timewindow per rest :
+              time_windows:
+                rest.timewindows.any? ?
+                  rest.timewindows.collect{ |tw|
+                    OrtoolsVrp::TimeWindow.new(start: tw.start, end: tw.end || 2**56)
```

Comment:
If the API lets in only one timewindow for rests, can we have a check here (to make sure that we didn't do anything incoherent in the code) and verify that `rest.timewindows.size <= 1` and raise otherwise. If not, the collect will happily send optim-ortools multiple timewindows and optim-ortools will just ignore them.